docs/developer-docs/web-apps/verifiable-credentials/relying-party.mdx

```diff
@@ -0,0 +1,327 @@
+        "origin": "https://employment-info.com",
+        "canisterId": "rwlgt-iiaaa-aaaaa-aaaaa-cai"
+    },
+    "credentialSpec": {
+        "credentialType": "VerifiedEmployee",
+        "arguments": {
+          "employerName": "XYZ Ltd."
+        }
+    },
+    "credentialSubject": "2mdal-aedsb-hlpnv-qu3zl-ae6on-72bt5-fwha5-xzs74-5dkaz-dfywi-aqe"
+  }
+}
+```
+
+In this request, the following parameters must be set:
+
+- `method`: `request_credential`: Uses the method for requesting a verifiable credential.
+
+- `params`: Configuration parameters to identify which credential is being requested.
+
+    - `issuer`: The information to identify which issuer API instance the request is sent to.
+
+        - `origin`: The origin URL of the issuer.
+
+        - `canisterId`: The canister ID of the issuer API. This field is optional.
+
+  - `credentialSpec`: The specification of the credential being requested.
+
+    - `credentialType`: The type of the requested credential.
+
+    - `arguments`: Key value pairs that are specific to the credential. This field is optional.
+
+  - `credentialSubject`: The principal of the user requesting the credential.
+
+  - `derivationOrigin`: An origin that should be used for the relying party's principal derivation as an alternative to the frontend hostname. This should match the `derivationOrigin` used when logging in the relying party.
+
+
+Additional examples can be found in the [verifiable credentials specification](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md#examples).
+
+### Step 3: Receive a response
+
+If the JSON RPC request is successfully sent and received by the issuer API, Internet Identity will return the following JSON RPC response to the relying party:
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "verifiablePresentation": "eyJQ..."
+  }
+}
+```
+
+In this response, `verifiablePresentation` is a JWT based verifiable presentation (VP) containing two credentials:
+
+1. A verifiable credential issued by Internet Identity that relates the `credentialSubject` to the user principal.
+
+2. A verifiable credential issued by the issuer to the user principal.
+
+The specification ensures unlinkability between the relying party and the issuer. That's why two credentials are needed. The first credential proves the link between the relying party's principal and `id_alias` created in the identity provider. The second credential is the actual credential to the `id_alias`.
+
+![Verifiable presentations](/static/img/docs/vcs-4.png)
+
+Note how the issuer's principal never appears in the credentials. Instead, the relying party and the issuer know about the `id_alias`. Therefore, this `id_alias` is also a credential to prove the link between the user's principal and the `id_alias`.
+
+The order of the credentials in the VP should match the order above, such that the credential relating to the `credentialSubject` to the user principal should come first.
```

Comment:
I find it a bit confusing. How about:

> The order of the credentials in the VP should match the order above. The `id_alias` credential linking with the relying party's principal should come first.

Reply:
Revised to "The order of the credentials in the VP should match the order above, such that the `id_alias` credential linking with the relying party's principal should come first."